build/Dockerfile

```diff
@@ -1,29 +1,21 @@
 # --- build smart gateway ---
-FROM centos:7 AS builder
+FROM registry.access.redhat.com/ubi8 AS builder
 ENV GOPATH=/go
 ENV D=/go/src/github.com/infrawatch/sg
 
 WORKDIR $D
 COPY . $D/
 
-RUN yum install epel-release -y && \
-        yum update -y --setopt=tsflags=nodocs && \
-        yum install git golang --setopt=tsflags=nodocs -y && \
-        yum clean all && \
+RUN dnf install golang git -y --setopt=tsflags=nodocs && \
         go build -o smart_gateway cmd/server/server.go && \
         mv smart_gateway /tmp/
 
 # --- end build, create smart gateway layer ---
-FROM centos:7
+FROM registry.access.redhat.com/ubi8
 
 LABEL io.k8s.display-name="Smart Gateway" \
       io.k8s.description="A component of the Service Telemetry Framework on the server side that ingests data from AMQP 1.x and provides a metrics scrape endpoint for Prometheus, and forwards events to ElasticSearch" \
-      maintainer="Leif Madsen <leif@redhat.com>"
-
-RUN yum install epel-release -y && \
```

Comment:
Do you happen to remember what was in epel that is now accounted for in UBI?

Reply:
I don't think we needed epel-release at all to be honest (or it was just a requirement to get golang or git). I suspect it is an artifact of the split between sg-bridge and sg-core, and that epel-release was providing the libraries for QPID (which is obviously not used here in sg-core).